ruby-challenges.rb

```diff
@@ -32,3 +60,18 @@
 # Expected output: 'LEARN is not a palindrome'
 palindrome_tester3 = 'Rotator'
 # Expected output: 'Rotator is a palindrome'
+
+
+def valid_palindrome(str)
+  if str.downcase == str.downcase.reverse
+    p "#{str} is a palindrome."
```

Comment:
Your logic looks great!  Oftentimes, best practice has the print when invoking the method and not inside the method itself.